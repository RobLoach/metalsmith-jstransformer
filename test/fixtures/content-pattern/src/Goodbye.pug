---
pageTitle: Goodbye World
pretty: true
---
doctype html
html(lang="en")
  head
    title= pageTitle
    script(type='text/javascript').
      if (foo) {
         bar(1 + 5)
      }
  body
    h1 Goodbye World!
