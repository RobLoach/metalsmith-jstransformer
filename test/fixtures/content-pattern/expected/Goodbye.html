<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Goodbye World</title>
    <script type="text/javascript">
      if (foo) {
         bar(1 + 5)
      }
    </script>
  </head>
  <body>
    <h1>Goodbye World!</h1>
  </body>
</html>